integration-tests/api/__tests__/admin/order-edit/ff-tax-inclusive-pricing.js

```diff
@@ -0,0 +1,192 @@
+const path = require("path")
+const { IdMap } = require("medusa-test-utils")
+
+const startServerWithEnvironment =
+  require("../../../../helpers/start-server-with-environment").default
+const { useApi } = require("../../../../helpers/use-api")
+const { useDb } = require("../../../../helpers/use-db")
+
+const adminSeeder = require("../../../helpers/admin-seeder")
+
+const {
+  simpleProductFactory,
+  simpleRegionFactory,
+  simpleCartFactory,
+} = require("../../../factories")
+
+jest.setTimeout(30000)
+
+const adminReqConfig = {
+  headers: {
+    Authorization: "Bearer test_token",
+  },
+}
+
+describe("[MEDUSA_FF_TAX_INCLUSIVE_PRICING] /admin/order-edits", () => {
+  let medusaProcess
+  let dbConnection
+
+  beforeAll(async () => {
+    const cwd = path.resolve(path.join(__dirname, "..", "..", ".."))
+    const [process, connection] = await startServerWithEnvironment({
+      cwd,
+      env: { MEDUSA_FF_TAX_INCLUSIVE_PRICING: true },
+    })
+    dbConnection = connection
+    medusaProcess = process
+  })
+
+  afterAll(async () => {
+    const db = useDb()
+    await db.shutdown()
+
+    medusaProcess.kill()
+  })
+
+  describe("Items totals", () => {
+    let product1
+    const prodId1 = IdMap.getId("prodId1")
+    const lineItemId1 = IdMap.getId("line-item-1")
+
+    beforeEach(async () => {
+      await adminSeeder(dbConnection)
+
+      product1 = await simpleProductFactory(dbConnection, {
+        id: prodId1,
+      })
+    })
+
+    afterEach(async () => {
+      const db = useDb()
+      return await db.teardown()
+    })
+
+    it("decorates items with (tax-inclusive) totals", async () => {
+      const taxInclusiveRegion = await simpleRegionFactory(dbConnection, {
+        tax_rate: 25,
+        includes_tax: true,
+      })
+
+      const taxInclusiveCart = await simpleCartFactory(dbConnection, {
+        email: "adrien@test.com",
+        region: taxInclusiveRegion.id,
+        line_items: [
+          {
+            id: lineItemId1,
+            variant_id: product1.variants[0].id,
+            quantity: 2,
+            unit_price: 10000,
+            includes_tax: true,
+          },
+        ],
+      })
+
+      const api = useApi()
+
+      await api.post(`/store/carts/${taxInclusiveCart.id}/payment-sessions`)
+
+      const completeRes = await api.post(
+        `/store/carts/${taxInclusiveCart.id}/complete`
+      )
+
+      const order = completeRes.data.data
+
+      const response = await api.post(
+        `/admin/order-edits/`,
+        {
+          order_id: order.id,
+          internal_note: "This is an internal note",
+        },
+        adminReqConfig
+      )
+
+      expect(response.status).toEqual(200)
+      expect(response.data.order_edit).toEqual(
+        expect.objectContaining({
+          items: expect.arrayContaining([
+            expect.objectContaining({
+              original_item_id: lineItemId1,
```

Comment:
thought: generally speaking, we have no idea where those amounts are coming from (it looks a bit like some magic numbers), my thought would be that we should instead have something like `item.quantity * item.unit_price` so that it makes sense to the reader without having to look at where this number is coming from. same for the taxes, something like `itemExpectedTaxes = 2000` and then again `quantity * itemExpectedTaxes`. wdyt? Not necessarily to change anything now but just a global thought

Reply:
I found, for myself, that this approach of derived calculation + a comment about quantity / unit price / tax makes a totals test easier to read but I agree we can introduce some "system" for expressing totlas going forward.